pkg/generate-sabakan-machines/cmd/root.go

```diff
@@ -75,7 +75,7 @@ Example:
 					datacenter, rackString, serial, product, role)
 			}
 
-			rack, err := strconv.ParseInt(rackString, 30, 64)
+			rack, err := strconv.ParseInt(rackString, 10, 64)
```

Comment:
I don't think `ParseInt()` is necessary because we don't need int64.

```suggestion
			rack, err := strconv.Atoi(rackString)
```